src/tests/Common/XUnitLogChecker/XUnitLogChecker.csproj

```diff
@@ -5,7 +5,9 @@
     <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PublishTrimmed>true</PublishTrimmed>
     <PublishAot Condition="'$(UseNativeAotForComponents)' == 'true'">true</PublishAot>
+    <PublishSingleFile Condition="'$(UseNativeAotForComponents)' != 'true'">true</PublishSingleFile>
```

Comment:
@jkoritzinsky, I am unable to build src/tests on riscv64 since this change was merged in main https://github.com/am11/CrossRepoCITesting/actions/runs/12436232286/job/34723677126
```
/runtime/.dotnet/sdk/10.0.100-alpha.1.24610.7/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(237,5): error NETSDK1067: Self-contained applications are required to use the application host. Either set SelfContained to false or set UseAppHost to true. [/runtime/src/tests/Common/XUnitLogChecker/XUnitLogChecker.csproj]
```

My workflow is same as @dotnet/samsung. We can either merge https://github.com/dotnet/runtime/pull/105004 and announce the new multistage worlflow for community platforms or (we cannot fast track that) revert https://github.com/dotnet/runtime/pull/110477/commits/9fc21e0f65be31a44ac27e9415aca3221c50b300.